datafusion/core/tests/parquet/page_pruning.rs

```diff
@@ -871,6 +871,55 @@ async fn without_pushdown_filter() {
     assert!(bytes_scanned_with_filter > bytes_scanned_without_filter);
 }
 
+#[tokio::test]
+// Data layout like this:
+// row_group1: page1(1~5), page2(All Null)
+// row_group2: page1(1~5), page2(6~10)
+// row_group3: page1(1~5), page2(6~9 + Null)
+// row_group4: page1(1~5), page2(All Null)
+// total 40 rows
+async fn test_pages_with_null_values() {
+    test_prune(
+        Scenario::WithNullValuesPageLevel,
+        "SELECT * FROM t where i8 <= 6",
+        Some(0),
+        // expect prune two pages which 10 rows
```

Comment:
I went through the tests carefully and made notes for myself of which row groups were being pruned. I figured I would suggest adding them as comments as well

```suggestion
        // expect prune pages with all null or pages that have only values greater than 6
        // (row_group1, page2), (row_group4, page2)
```